pyproject.toml

```
@@ -51,9 +51,10 @@ test = [
 
 [build-system]
 requires = [
-    "setuptools>=30.3",
-    "setuptools_scm[toml]>=3.4",
-    "oldest-supported-numpy; python_version>='3.7'",
+    "setuptools>=61.2",
+    "setuptools_scm[toml]>=3.6",
+    "numpy>=1.25,<2",
```

```suggestion
    "numpy",
```

You can't test with NumPy 2.x (devdeps CI) if you max pin NumPy.